yt_dlp/extractor/cnn.py

```diff
@@ -141,3 +141,56 @@ def _real_extract(self, url):
         webpage = self._download_webpage(url, url_basename(url))
         cnn_url = self._html_search_regex(r"video:\s*'([^']+)'", webpage, 'cnn url')
         return self.url_result('http://cnn.com/video/?/video/' + cnn_url, CNNIE.ie_key())
+
+
+class CNNIndonesiaIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.cnnindonesia\.com/[\w-]+/(?P<upload_date>\d{8})\d+-\d+-(?P<id>\d+)/(?P<display_id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://www.cnnindonesia.com/ekonomi/20220909212635-89-845885/alasan-harga-bbm-di-indonesia-masih-disubsidi',
+        'info_dict': {
+            'id': '845885',
+            'ext': 'mp4',
+            'description': 'md5:e7954bfa6f1749bc9ef0c079a719c347',
+            'upload_date': '20220909',
+            'title': 'Alasan Harga BBM di Indonesia Masih Disubsidi',
+            'timestamp': 1662859088,
+            'duration': 120.0,
+            'thumbnail': r're:https://akcdn\.detik\.net\.id/visual/2022/09/09/thumbnail-ekopedia-alasan-harga-bbm-disubsidi_169\.jpeg',
+            'tags': ['ekopedia', 'subsidi bbm', 'subsidi', 'bbm', 'bbm subsidi', 'harga pertalite naik'],
+            'age_limit': 0,
+            'release_timestamp': 1662859088,
+            'subtitle': {},
```

Comment:
?

Reply:
I forgot this one, fixed in latest commit